k3s/testcontainers/k3s/__init__.py

```diff
@@ -53,8 +54,11 @@ def start(self) -> "K3SContainer":
         return self
 
     def config_yaml(self) -> str:
-        output = self.get_wrapped_container().exec_run(['cat', '/etc/rancher/k3s/k3s.yaml'])
-        config_yaml = output.output.decode('utf-8') \
-            .replace('https://127.0.0.1:{}'.format(self.KUBE_SECURE_PORT), 'https://localhost:{}'
-                     .format(self.get_exposed_port(self.KUBE_SECURE_PORT)))
+        """This function returns the kubernetes config yaml which can be used
+        to initialise python client
+        """
+        execution = self.get_wrapped_container().exec_run(['cat', '/etc/rancher/k3s/k3s.yaml'])
+        config_yaml = execution.output.decode('utf-8') \
+            .replace(f'https://127.0.0.1:{self.KUBE_SECURE_PORT}',
+                     f'https://localhost:{self.get_exposed_port(self.KUBE_SECURE_PORT)}')
```

Comment:
should we replace this as well with ? self.get_container_host_ip()

Reply:
apologies, this one remained. Fixing now.